tests/framework/Optimizers/GradientBasedOptimizers/t.xml

```diff
@@ -1,25 +1,20 @@
 <?xml version="1.0" ?>
```

Comment:
why is this test named "t" and replacing the max_range_projectile_norm_grad test?